presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -73,23 +87,37 @@ public void setup()
         server.createCatalog(TEST_CATALOG, "tpch");
         server.installPlugin(new BlackHolePlugin());
         server.createCatalog("blackhole", "blackhole");
-
+        waitForNodeRefresh(server);
         setupTestTables();
     }
 
+    private static void waitForNodeRefresh(TestingPrestoServer server)
```

Comment:
Unfortunately it is necessary. Without it jdbc cancellation test fails sporadically if i run it as a single test.